src/crystal_lib/lib_transformer.cr

```diff
@@ -77,12 +78,32 @@ class CrystalLib::LibTransformer < Crystal::Transformer
           else
             raise "Include remove_prefix value must be a bool literal, at #{value.location}"
           end
+        when "import_docstrings"
+          value = named_arg.value
+          case value
```

Comment:
This could probably be more concise when all variants are combined in a single case expression:

```cr
case value.value
when "brief" ...
when "full", true ...
when "none", false
else ...
end
```

Reply:
Good idea, I'm going to change that :)